### ISSUE TITLE
Using Sickle on Grass in protected region produces infinite Hay

### ISSUE THREAD

**Issue by Ijwu** [NyxStudios/TShock#1442]
![image](https://cloud.githubusercontent.com/assets/1393017/25825789/41b6b596-3412-11e7-8bba-2ae975646b42.png)

May be a recurrence of https://github.com/NyxStudios/TShock/issues/698. However, I am not satisfied with the solution of this issue. Is there something we can do to stop the garnering of infinite hay?

**Comment by AndrioCelos** [comment]
For what it's worth, you can get the same effect with flower boots.

**Comment by mooninitevii** [comment]
As long as spawn protection is on and player does not have permission tshock.world.editspawn, if player uses item 3017 (flower boots) on grass in the spawn radius, any player will be able to cut unlimited grass in that spot with or without item 3017.
That is until, and this is specific, a player with permission tshock.world.editspawn cuts grass block in that spot, at which point the ghost grass no longer exists.

TL;DR
It only starts to happen when someone with out the spawnedit perm uses the flower boots in a protected spawn. Once the "ghost grass" grows you don't need flower boots to keep making it happen.
